products/compute/examples/chef/address.rb

```diff
@@ -21,16 +21,10 @@
 <%= compile 'templates/chef/example~auth.rb.erb' -%>
 
 <% end -%>
-gcompute_region <%= example_resource_name('some-region') -%> do
-  action :create
-  r_label 'us-west1'
-  project ENV['PROJECT'] # ex: 'my-test-project'
-  credential 'mycred'
-end
 
 gcompute_address <%= example_resource_name('test1') -%> do
   action :create
-  region <%= example_resource_name('some-region') %>
+  region <%= example_resource_name('us-west1') %>
```

Comment:
We shouldn't use the example_resource_name function in this case.

A cookbook's recipe folder has two different sets of recipes: a "examples~*" set and a "tests~*" set.

Examples are examples, tests are for a small integration test runner that we haven't used in a very, very long time. (Our new testing strategy will make these tests files unnecessary)

Both of them are copies of these example files. The only difference lies in the `example_resource_name` function. On examples, using `example_resource_name` does nothing. On tests, it adds in the prefix 'chef-e2e'. This is so we can run all of the tests and be reasonably certain that none of resources it creates will already be in our GCP project. (TLDR; tech debt).

In this example, the examples file will show `us-west1` which is a valid GCP region and fine. On the tests file, the file will show `chef-e2e-us-west1` which isn't a valid GCP region and will crash.

Reply:
(Please change on all of them)

Reply:
Didn't know about that. Reverted those changes.